app/managers.py

```diff
@@ -0,0 +1,51 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")
+
+    def all(self) -> list:

```

Comment:
It would be better to indicate that you return a list of Actor instances, not just list

Reply:
Yes . I totally agree with you.